scss/components/_table-styles.scss

```diff
@@ -253,6 +253,84 @@ h3 + .simple-table {
   td:first-child {
     border-right: 1px solid $gray-medium;
     padding-left: u(4rem);
-    font-style: italic;
+     font-style: italic;
   }
 }
+
+// For custom_table_block in streamfactory
+
+.custom-table {
+  box-shadow: -8px 0px 9px -2px $gray-medium inset;
+  overflow-x: scroll;
+  display: block;
+  margin-bottom: u(1rem);
+
+  table {
+    @extend .simple-table;
+    width: auto;
+
+    th {
+      @extend .simple-table__header;
+      table-layout: auto;
+      padding: u(1rem);
+      text-align: left;
+      border-right: 1px solid $gray-medium;
+    }
+
+    tr {
+      @extend .simple-table__row;
+      th:first-child {
+        border-bottom: 1px solid $neutral;
+      }
+    }
+
+    td {
+      @extend .simple-table__cell;
+      border-right: 1px solid $gray-medium;
+      text-align: right;
+      padding: u(1rem) !important;
+    }
+
+    td:last-child,
```

Comment:
You can get rid of these by unnesting all of these from `table` and just having `.cucstom-table { th { } }` etc. Would that work with the styling?